sdks/java/maven-archetypes/examples/build.gradle

```diff
@@ -17,10 +17,44 @@
  */
 
 apply from: project(":").file("build_rules.gradle")
-applyJavaNature()
+applyJavaNature(artifactId: "beam-sdks-java-maven-archetypes-examples")
 
 description = "Apache Beam :: SDKs :: Java :: Maven Archetypes :: Examples"
 
+processResources {
+   filter org.apache.tools.ant.filters.ReplaceTokens, tokens: [
+      'project.version':  version,
+      'bigquery.version': 'v2-rev374-1.22.0',
```

Comment:
instead of hardcoding this please use the version from the library.java.dependency defined in build.gradle?

Reply:
Extracting the versions from the project.library.java.* libraries. I did not see maven plugin versions defined elsewhere, so those are still hardcoded.